tools/cli/go-whisk/whisk/client.go

```diff
@@ -293,7 +295,12 @@ func (c *Client) Do(req *http.Request, v interface{}) (*http.Response, error) {
     // Handle 1. HTTP Success + Valid body matching request expectations
     // Handle 3. HTTP Success + Body does NOT match request expectations
     if IsHttpRespSuccess(resp) && v != nil {
-        return parseSuccessResponse(resp, data, v), nil
+        if resp.StatusCode == 202 {
+            err = MakeWskError(errors.New("Time limit exceeded."), 202, NO_DISPLAY_MSG, NO_DISPLAY_USAGE,
+                NO_MSG_DISPLAYED)
+        }
+
+        return parseSuccessResponse(resp, data, v), err
```

Comment:
@mdeuser, I can only speak for action invocations. They return only an activation ID.